storio-common/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'checkstyle'
+apply plugin: 'com.uber.infer.android'
```

Comment:
Maybe apply it in root `build.gradle` for all `android` subprojects?

See https://github.com/pushtorefresh/storio/blob/592368c97524f736e315d7e89cf4f2b339c657b7/build.gradle#L107